src/modules/app/components/Layout.css

```diff
@@ -1,10 +1,6 @@
 @import url('../../common/constants/colors.css');
 @import url('../../common/constants/fonts.css');
 
-.top-spacing {
-  padding-top: 22px;
-}
-
 .content {
```

Comment:
Since this removes the space between Header and breadcrumbs, the breadcrumbs now stick to the header. But now you could add sth. like `padding: 10px 0;` to the Breadcrumbs container, which is good anyway, because that way we increase the touch-target for scrolling the breadcrumbs when they're too long.

Reply:
i think we never have breadcrumbs as part of Layout, just as part of RichLayout

Reply:
ok, you were right, done :D